cms/analysis/tests/test_models.py

```diff
@@ -31,6 +31,15 @@ def test_index_redirects_to_latest(self):
         response = self.client.get(self.series.url)
         self.assertRedirects(response, self.series.url + self.series.reverse_subpage("latest_release"))
 
+    def test_index_redirects_to_latest_in_external_env(self):
```

Comment:
Q: Is there something we can do to ensure, all external pages render when external true vs devs having to remember to add a test case with external true as pages get added.

Reply:
Possibly, but it'd be quite brittle. I could try and discover pages, their factories and attempt to render them, but that would be quite brittle for more complex pages.